### DISCUSSION TITLE
[UPLOAD FAILED] Premature close

### DISCUSSION THREAD

**Discussion by DemonAngeX2** [immich-app/immich#3182] [Q&A]
### The bug

Hello, i tried to upload a .MOV video but i have this error in the docker container:

Error [ERR_STREAM_PREMATURE_CLOSE]: Premature close
    at new NodeError (node:internal/errors:399:5)
    at ServerResponse.onclose (node:internal/streams/end-of-stream:154:30)
    at ServerResponse.emit (node:events:525:35)
    at emitCloseNT (node:_http_server:984:10)
    at Socket.onServerResponseClose (node:_http_server:277:5)
    at Socket.emit (node:events:525:35)
    at TCP.<anonymous> (node:net:322:12)

### The OS that Immich Server is running on

Unraid (docker)

### Version of Immich Server

immich:v1.62.1-ig127

### Version of Immich Mobile App

latest IOS

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
ghcr.io/imagegenius/immich
```

### Your .env content

```Shell
ghcr.io/imagegenius/immich
```

### Reproduction steps

```bash
I tried to upload a MOV video file. 
Resolution: 4K
Size: 151Mo
Time: 14s

Second file wich failed: 
Name: IMG_0060.MOV
Size:162
Time: 15s
```

**Comment by etnoy** [comment]
How did you initiate the upload? Mobile? Web? Cli?

**Comment by bo0tzz** [comment]
The error you posted happens when you stop viewing a video in the web UI and can be ignored. Are there any other errors in the logs?

**Comment by DemonAngeX2** [comment]
> How did you initiate the upload? Mobile? Web? Cli?

Web

**Comment by DemonAngeX2** [comment]
> The error you posted happens when you stop viewing a video in the web UI and can be ignored. Are there any other errors in the logs?

I updated to new version and no suspicious log. I retried to upload with new version but it's the same .... The only thing is a warning to set sysctl vm.overcommit_memory=1. I did it. I will try if it can solve the problem after a reboot.

**Comment by DemonAngeX2** [comment]
Hello, can i have help on this problem. I reencoded both videos. Converted in x265 10 bits 4K MP4

Before it was a MOV in 4K took with my iphone.

I tried to import them and on video successfully imported. The imported video was converted from 151Mo to 30Mo. The failed import video 162 Mo to 105 Mo

It said in documentation logs are in docker container but i have no error....

**Comment by baranaltay** [comment]
Hello, I have the issue as well for a long time. 
### Properties
**Host platform**: Raspberry Pi 4 8gb
**Mobile platform**: iOS 16
**Device**: iPhone 13 Pro
**Video**: 02:49 minutes of 4k@60 HDR Dolby Vision HEVC
**Size**: 2,24 GB
**Immich server version**: 1.83
**Immich mobile version**: 1.83

### Info screenshot of the asset:
![img](https://github.com/immich-app/immich/assets/34520068/49168bbf-f946-44b1-9627-e2f16426bb9c)

### Error log on the server side
```[Nest] 8  - 11/01/2023, 10:02:45 PM   ERROR [ExpressAdapter] Premature close
Error [ERR_STREAM_PREMATURE_CLOSE]: Premature close
    at new NodeError (node:internal/errors:406:5)
    at ServerResponse.onclose (node:internal/streams/end-of-stream:159:30)
    at ServerResponse.emit (node:events:526:35)
    at emitCloseNT (node:_http_server:1023:10)
    at Socket.onServerResponseClose (node:_http_server:278:5)
    at Socket.emit (node:events:526:35)
    at TCP.<anonymous> (node:net:337:12)
```

### Behavior
When the auto-backup functionality kicks in or I manually open this asset from immich app and click upload; it starts, slowly progresses, stucks at 100% and then tries again or moves on to the next asset. Wi-Fi speed is around 300 - 500 megabit/s.

I can provide more information or perform additional steps if requested. Just wanted to help. I love what you guys are doing here! ❤
(edit: added immich versions)
(edit2: added host info)

**Reply by baranaltay** [reply]
P.S.: It works when I use the CLI tool. It takes its time but gets the job done.
